Manifests/MS/WinPS-Manifest.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Company: Microsoft; Product: Windows Powershell (powershell.exe) and Powershell Core (pwsh.exe) -->
+<AppManifest Id="Powershell" xmlns="urn:schemas-microsoft-com:windows-defender-application-control">
+    <!--
+        When specified, scripts will be blocked from executing on
+        policy failure vs being allowed to run in Constrained
+        Language Mode.
+
+        (WldpCanExecute* family will return WLDP_EXECUTION_POLICY_BLOCKED).
+
+        Supported on both PSCore and Windows PowerShell
+    -->
+    <SettingDefinition
+        Name="BlockScriptOnPolicyFailure"
+        Type="Bool"
+        IgnoreAuditPolicies="false"

```

Comment:
@jgeurten should we be ignoring audit policies here? it's ultimately only affecting an enforcement action, so it seems like we should maybe ignore.

@jsuther1974 any thoughts?

Reply:
Chatted with Deven offline and agree that IgnoreAuditPolicies should be "true"

Reply:
Resolved offline. Setting this to 'true' makes it this behavior:

| Policy 1 | Policy 2 | Script that Passes Policy | Script that Fails Policy | 
|:---------|:---------|:---------------------------|:-------------------------|
| Audit Policy, No App Settings | Enforce Policy, No App Settings | FullLanguage | Constrained Language |
| Audit Policy, BlockScriptsOnFailure | Enforce Policy, No App Settings | FullLanguage | Constrained Language |
| Audit Policy, No App Settings | Enforce Policy, BlockScriptsOnFailure | FullLanguage | No Execute |
| Audit Policy, BlockScriptsOnFailure | Enforce Policy, BlockScriptsOnFailure | FullLanguage | No Execute |
| Audit Policy, No App Settings |  | Full Language | Full Language |
| Audit Policy, BlockScriptsOnFailure|  | Full Language | Full Language |
| Enforce Policy, No App Settings|  | Full Language | Constrained Language |
| Enforce Policy, BlockScriptsOnFailure|  | Full Language | No Execute |

As a reminder, per the Boolean resolution strategy across multiple policies, all policy groups must set this value to 'true' for it to evaluate to 'true'.